meeshkan/serve/mock/faker/stateless_faker.py

```diff
@@ -0,0 +1,323 @@
+import json
+import random
+import typing
+from dataclasses import dataclass
+from functools import reduce
+from typing import Any, Mapping, Sequence, Union, cast
+
+from http_types import Request, Response
+from openapi_typed_2 import OpenAPIObject, Reference, Schema, convert_from_openapi
+
+from meeshkan.serve.mock.faker.faker_base import FakerBase
+from meeshkan.serve.mock.faker.faker_exception import FakerException
+from meeshkan.serve.mock.matcher import (
+    change_ref,
+    change_refs,
+    get_response_from_ref,
+    ref_name,
+)
+from meeshkan.serve.mock.storage.entity import Entity
+from meeshkan.serve.mock.storage.mock_data import MockData
+from meeshkan.serve.utils.opanapi_utils import get_x
+
+
+@dataclass
+class FakerData:
```

Comment:
docstring would be nice, I think. What is this for? Who uses this? What's the single entity in `FakerData`? What's path_item string?

Reply:
Added comments for that